Content.Client/GameObjects/EntitySystems/RangedWeaponSystem.cs

```diff
@@ -78,7 +78,6 @@ public override void Update(float frameTime)
 
             if (weapon.Automatic || canFireSemi)
             {
-                Logger.Debug(IoCManager.Resolve<IGameTiming>().CurTick.ToString());
```

Comment:
I removed this line since it spammed the hell out of my console when firing guns.

Reply:
Blame PJB for leaving it in.